libs/execution_context/ebpf_maps.c

```diff
@@ -746,6 +746,10 @@ _delete_hash_map(_In_ _Post_invalid_ ebpf_core_map_t* map)
 static void
 _delete_object_hash_map(_In_ _Post_invalid_ ebpf_core_map_t* map)
 {
+    if (!map) {
+        return;
+    }
```

Comment:
Why is this needed?  Since the SAL annotation has _In_ it should be the caller's responsibility to check this.
If this code path could ever be hit, it means that the caller should have already seen an error due to the SAL annotation.

Reply:
Failed map allocation triggers an AV here.

Reply:
I don't doubt that. But as is, this is not the correct fix.  Line 747 says the contract is that the caller's responsibility is to never call it with null.  So unless you change the contract for all delete_entry() functions, then the check here needs to be moved out to the caller so that this function is never invoked with a null.